eng/deploy.yaml

```diff
@@ -0,0 +1,148 @@
+parameters:
+  Subscription: ''
+  ServiceFabricConnection: ''
+  PublishProfile: ''
+  DotNetStatusAppName: ''
+  MaestroUrl: ''
+variables:
+  Subscription: ${{ parameters.Subscription }}
+  ServiceFabricConnection: ${{ parameters.ServiceFabricConnection }}
+  PublishProfile: ${{ parameters.PublishProfile }}
+  DotNetStatusAppName: ${{ parameters.DotNetStatusAppName }}
+  MaestroUrl: ${{ parameters.MaestroUrl }}
+
+  # --- Secret Variable group requirements ---
+  # maestro-admin-sql-connection-string
+  # maestro-token
+  # dn-bot-dnceng-build-rw-code-rw-release-rw
+  # maestro-scenario-test-github-token
+
+stages:
+- stage: deploy
+  displayName: Deploy
+  dependsOn: build
+  jobs:
+  - job: updateDatabase
+    displayName: 
+    steps:
+    - download: current
+      artifact: ReleaseUtilities
+    - download: current
+      artifact: Maestro.Data
+    - task: AzurePowerShell@3
+      displayName: eng\firewall.ps1 -Add
+      inputs:
+        azureSubscription: ${{ parameters.Subscription }}
+        ScriptPath: $(Pipeline.Workspace)/ReleaseUtilities/firewall.ps1
+        ScriptArguments: -RuleName UnblockSQLForUpgrade -Add -ConnectionString "$(maestro-admin-sql-connection-string)"
+        azurePowerShellVersion: LatestVersion
+    - task: PowerShell@2
+      displayName: update-database.ps1
+      inputs:
+        targetType: filePath
+        filePath: $(Pipeline.Workspace)/Maestro.Data/update-database.ps1
+        arguments: Maestro.Data.dll
+        workingDirectory: $(Pipeline.Workspace)/Maestro.Data
+      env:
+        BUILD_ASSET_REGISTRY_DB_CONNECTION_STRING: $(maestro-admin-sql-connection-string)
+    - task: AzurePowerShell@3
+      displayName: eng\firewall.ps1 -Remove
+      inputs:
+        azureSubscription: ${{ parameters.Subscription }}
+        ScriptPath: $(Pipeline.Workspace)/ReleaseUtilities/firewall.ps1
+        ScriptArguments: -RuleName UnblockSQLForUpgrade -Remove -ConnectionString "$(maestro-admin-sql-connection-string)"
+        azurePowerShellVersion: LatestVersion
+      condition: always()
+
+  - job: deployMaestro
+    displayName: Deploy maestro service fabric application
+    dependsOn: updateDatabase
+    steps:
+    - download: current
+      artifact: MaestroApplication
+    - task: ServiceFabricDeploy@1
+      displayName: Deploy Service Fabric Application (Darc)
```

Comment:
>Darc [](start = 54, length = 4)

Why does this say Darc? its not deploying darc, its deploying Maestro

Reply:
That's what it said before. I'll remove it. :-)

---
In reply to: [344300953](https://github.com/dotnet/arcade-services/pull/756#discussion_r344300953) [](ancestors = 344300953)